packages/search-in-workspace/src/browser/search-in-workspace-frontend-contribution.ts

```diff
@@ -376,9 +408,17 @@ export class SearchInWorkspaceFrontendContribution extends AbstractViewContribut
             order: '2'
         });
         menus.registerMenuAction(CommonMenus.EDIT_FIND, {
-            commandId: SearchInWorkspaceCommands.REPLACE_IN_FILES.id,
+            commandId: SearchInWorkspaceCommands.FOCUS_NEXT_RESULT.id,
             order: '3'
         });
+        menus.registerMenuAction(CommonMenus.EDIT_FIND, {
+            commandId: SearchInWorkspaceCommands.FOCUS_PREV_RESULT.id,
+            order: '4'
+        });
+        menus.registerMenuAction(CommonMenus.EDIT_FIND, {
+            commandId: SearchInWorkspaceCommands.REPLACE_IN_FILES.id,
+            order: '5'
+        });
```

Comment:
I believe changes to the global `edit` menu should be reverted. I don't see the same commands present in vscode's menu, and downstream adopters might not expect it. They can always add it themselves if they wish.